File: tests/rustdoc-js/merged-doc.js
const EXPECTED = [
    {
        'query': 'merged_doc::Doc',
        'others': [
            { 'path': 'merged_doc', 'name': 'Doc' },
        ],
    },
    {
        'query': 'merged_dep::Dep',
        'others': REVISION === "nomerge" ? [] :
            [
                { 'path': 'merged_dep', 'name': 'Dep' },
            ],
    },
];

File: tests/rustdoc-js/merged-doc.rs
//@ revisions: merge nomerge
//@ aux-build:merged-dep.rs
//@ build-aux-docs
//@[merge] doc-flags:--merge=finalize
//@[merge] doc-flags:--include-parts-dir=info/doc.parts/merged-dep
//@[merge] doc-flags:-Zunstable-options

extern crate merged_dep;

pub struct Doc;
